docs/db/README.md

```diff
@@ -1,6 +1,6 @@
 <!--- Hugo front matter used to generate the website version of this page:
 linkTitle: Database
-aliases: [/docs/specs/semconv/database]
+redirects: [{ from: 'database/*', to: ':splat' }]
```

Comment:
@vitorvasc - let's give this one last try:

```suggestion
redirects: [{ from: '../database/*', to: ':splat' }]
```

The redirects might not yet have full relative-path support yet.